package.json

```diff
@@ -5,7 +5,9 @@
   "main": "./lib/main.js",
   "scripts": {
     "lint": "eslint . --ext .ts",
-    "test": "nyc ts-mocha --no-deprecation -t 60000 --exit test/**/*.ts",
+    "test": "nyc ts-mocha --no-deprecation -t 60000 --exit test/**/*.ts test/**/**/*.ts",
+    "echo": "echo \"test/**/*.ts\"",
+    "echo2": "echo test/**/*.ts",
```

Comment:
echo2 sounds like a placeholder. Why do we need an echo command?

Reply:
Those can be removed. Fixed in the new commit.